src/styles/header.scss

```diff
@@ -0,0 +1,66 @@
+@use './variables' as *;
+
+.moyo-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+  padding: 0 50px;
+  overflow: auto;
+
+  &__logotype {
+    width: $logo-header-size;
+    height: $logo-header-size;
+    margin-right: 40px;
+  }
+
+  &__list {
+    display: flex;
+    align-items: center;
+  }
+
+  &__item {
+    margin-right: 20px;
+    font-family: Roboto, sans-serif;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), you should avoid specifying the font-family multiple times if it is the same throughout the project. You can set the font-family once in the `html` or `body` selector.